ldaptor/protocols/pureber.py

```diff
@@ -225,7 +225,7 @@ def toWire(self):
         return result
 
     def __repr__(self):
-        value = repr_converter(self.value)
+        value = self.value
```

Comment:
this is ok... but maybe have a single representation for all instances

I have no idea why we need a special case here.

```
return self.__class__.__name__ + "(value=%r, tag=%d)" % (value, self.tag)
```

Reply:
probably going to attrs everything next week, dunno why tag is sometimes excluded.